docs/extensions/provider-local.md

```diff
@@ -35,6 +35,10 @@ Please note that all of them are no technical limitations/blockers but simply ad
    _When [`APIServerSNI`](../proposals/08-shoot-apiserver-via-sni.md) is disabled then gardenlet uses load balancer `Service`s in order to expose the shoot clusters' `kube-apiserver`s. Typically, local Kubernetes clusters don't support this. In this case, the local extension uses the host IP to expose the `kube-apiserver`, however, this can only be done once._\
    _However, given that the `APIServerSNI` feature gate is deprecated and will be removed in the future (see [gardener/gardener#6007](https://github.com/gardener/gardener/pull/6007)), we will probably not invest into this._
 
+5. In case a seed cluster with multiple availability zones, i.e. multiple entries in `.spec.provider.zones`, is used in conjunction with a single-zone shoot control plane, i.e. a shoot cluster without `.spec.controlPlane.highAvailability` or with `.spec.controlPlane.highAvailability.faultTolerance.type` set to `node`, the local address of the api server endpoint needs to be determined manually or via the in cluster `coredns`.
+
+   _As the different istio ingress gateway loadbalancers have individual external IP addresses, single-zone shoot control planes can end up in a random availability zone. Having the local host use the `coredns` in the cluster as name resolver would form a name resolution cycle. The tests mitigate the issue by adapting the DNS configuration inside the affected test._
+
```

Comment:
I am not sure I understand your suggestion. Do you want to schedule all single-zone shoot control planes in the local scenario to the first zone? This would indeed make the scenario easier as it would be less dynamic, but it would also more diverge from the real-world scenario.
The upgrade zone test would still need to cope with a dynamic switch as the domain name will switch from one istio load balancer to another one. Therefore, I am not sure if we will really improve considerably if pinning all single-zone shoot control planes to the first zone.